test-cases-internal/src/main/java/com/palantir/conjure/verification/GenerateServerServices.java

```diff
@@ -96,14 +117,22 @@ private static void writeServiceDefinition(
     private static Map<String, Object> generateAutoDeserializeService(List<BodyTests> bodyTests) {
         ImmutableMap.Builder<String, Object> endpoints = ImmutableMap.builder();
 
-        bodyTests.stream().map(BodyTests::getType).map(TestCasesUtils::parseConjureType).forEach(conjureType -> {
-            String endpointName = TestCasesUtils.typeToEndpointName("receive", conjureType);
-            String typeName = conjureType.visit(new ResolveLocalReferencesConjureTypeVisitor());
-            endpoints.put(endpointName, ImmutableMap.of(
-                    "http", "GET /" + endpointName + "/{index}",
-                    "returns", typeName,
-                    "args", ImmutableMap.of("index", "integer")));
-        });
+        bodyTests.stream()
+                .map(BodyTests::getType)
+                .map(TestCasesUtils::parseConjureType)
+                .forEach(conjureType -> {
+                    String endpointName = TestCasesUtils.typeToEndpointName("receive", conjureType);
+                    String typeName = conjureType.visit(new ResolveLocalReferencesConjureTypeVisitor());
+                    endpoints.put(
+                            endpointName,
+                            ImmutableMap.of(
+                                    "http",
+                                    "GET /" + endpointName + "/{index}",
```

Comment:
This looks like a regression @dansanduleac

Reply:
I think this might be hitting some obscure edge case that tries to assert that N/2+1 rows have the same "kind" of tree in the AST.
The values here seem to be 
* Kind.PLUS (BinaryTree of `+` operations)
* Kind.VARIABLE
* Kind.METHOD_INVOCATION

https://github.com/palantir/palantir-java-format/blob/fc94362438d9450b2fefe29315e5ea687bd93215/palantir-java-format/src/main/java/com/palantir/javaformat/java/JavaInputAstVisitor.java#L3222-L3226

I honestly don't see much reason for this kind of heuristic, so it might make sense to remove it.